pkg/apis/operator/v1alpha1/validation/validation.go

```diff
@@ -415,7 +415,7 @@ func validateGardenerDashboardConfig(config *operatorv1alpha1.GardenerDashboardC
 	}
 
 	if oidc := config.OIDC; oidc != nil {
-		oidcPath := fldPath.Child("oidc")
+		oidcPath := fldPath.Child("oidcConfig")
```

Comment:
Can we rename the golang field name to OIDCConfig? K8s code-generator should generate a violation/warning when the golang field name does not match the json field name.

Reply:
Yes